common/constant/env.go

```diff
@@ -21,4 +21,6 @@ const (
 	CONF_CONSUMER_FILE_PATH        = "CONF_CONSUMER_FILE_PATH"
 	CONF_PROVIDER_FILE_PATH        = "CONF_PROVIDER_FILE_PATH"
 	APP_LOG_CONF_FILE       string = "APP_LOG_CONF_FILE"
+	DUBBO_VERSION_KEY              = "dubbo"
+	DUBBO_VERSION                  = "2.0.2"
```

Comment:
"2.0.2" dubbo version? does it exist?

Reply:
`response attachmentds` will be validated whether it is supported in dubbo server. And when dubbo protocol version is `2.0.10~2.62`, it is not supported. Refer to https://github.com/apache/dubbo/blob/a36cc7520e0150d07d30e8baa0c61d5f9f11e6ed/dubbo-common/src/main/java/org/apache/dubbo/common/Version.java#L96

Reply:
Maybe you should place it into the code as a comment.

Reply:
yes, I am doing it.